src/seedu/addressbook/AddressBook.java

```diff
@@ -308,11 +308,20 @@ private static void setupDefaultFileForStorage() {
      * Returns true if the given file is acceptable.
      * The file path is acceptable if it ends in '.txt'
      * TODO: Implement a more rigorous validity checking.
+     * Note: Access modifier was changed from private to 
```

Comment:
The comments and method name could have mislead you to misinterpret the purpose of the method, in which case we should update them too.